src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+
+  &__list {
+    display: flex;
+    margin-right: 50px;
+    list-style: none;
+  }
+
+  &__item {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    text-transform: uppercase;
+  }
+
+  &__link {
+    position: relative;
+    display: inline-block;
+    padding: 20px 0;
+    margin-left: 20px;
+    transition: color 0.3s;
+    text-decoration: none;
+    color: $color-primary;
+    cursor: pointer;
+  }
+
+  &__link:hover {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 10th point